ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/base-journey/services/journey-routing-listener.service.spec.ts

```diff
@@ -100,7 +94,7 @@ describe('JourneyRoutingListenerService', () => {
         givenCurrentUrlIs('/login');
         service.startRouting(bindings, journey);
 
-        const rootUrl = `/${AppPaths.Root}`;
+        const rootUrl = `/${AppPaths.JourneySelector}`;
```

Comment:
Name of this test says "if routed to application home" but this goes to JourneySelector. Should update to be more clear and change const name.

Reply:
The root wasn't a good name, to begin with. I wanted it to be clear what the default component is and also the start for all journeys